cmd/tnf/claim/compare/compare.go

```diff
@@ -7,30 +7,39 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/test-network-function/cnf-certification-test/cmd/tnf/pkg/claim"
 )
 
-type claimFileStruct struct {
-	Claim struct {
-		Nodes struct {
-			CniPlugins  map[string][]Cni       `json:"cniPlugins"`
-			NodesHwInfo map[string]interface{} `json:"nodesHwInfo"`
-			CsiDriver   interface{}            `json:"csiDriver"`
-		} `json:"nodes"`
-
-		RawResults struct {
-			Cnfcertificationtest struct {
-				Testsuites struct {
-					Testsuite struct {
-						Testcase []testCase `json:"testcase"`
-					} `json:"testsuite"`
-				} `json:"testsuites"`
-			} `json:"cnf-certification-test"`
-		} `json:"rawResults"`
-	} `json:"claim"`
-}
-type Cni struct {
-	Name    string        "json:\"name\""
-	Plugins []interface{} "json:\"plugins\""
+var (
+	Claim1 string
+	Claim2 string
+
+	claimCompareFiles = &cobra.Command{
+		Use:   "compare",
+		Short: "Compare two claim files.",
+		RunE:  claimCompare,
+	}
+)
+
+func NewCommand() *cobra.Command {
+	claimCompareFiles.Flags().StringVarP(
+		&Claim1, "claim1", "1", "",
+		"existing claim1 file. (Required) first file to compare",
+	)
+	claimCompareFiles.Flags().StringVarP(
+		&Claim2, "claim2", "2", "",
+		"existing claim2 file. (Required) second file to compare with",
```

Comment:
```suggestion
		"existing claim2 file. (Required) second file to compare",
```

Match the other string?

Reply:
Fixed in https://github.com/test-network-function/cnf-certification-test/pull/1249/commits/0d669a36eddc5a5e186fed2ccbb63390082b84ff. Please note that most of the code of the compare subcommand was left as it was before. I just made the minimum changes to adapt the command/subcommand tree loading to the new folders structure.